src/train.jl

```diff
@@ -308,3 +308,17 @@ function sciml_train(loss, θ, opt::Optim.AbstractConstrainedOptimizer,
            Optim.Options(extended_trace=true,callback = _cb,
                          f_calls_limit = maxiters))
 end
+
+
+function sciml_train(loss, opt::Symbol = :adaptive_de_rand_1_bin, data = DEFAULT_DATA;lower_bounds, upper_bounds,
+                      maxiters = get_maxiters(data), kwargs...)
+  local x, cur, state
+  cur,state = iterate(data)
+
+  _loss = function (θ)
+    x = loss(θ,cur...)
+    first(x)
+  end
+
+  BlackBoxOptim.bboptimize(_loss;Method = opt, SearchRange = [(lower_bounds[i], upper_bounds[i]) for i in 1:length(lower_bounds)], kwargs...)

```

Comment:
@ChrisRackauckas This should then be converted to an Optim.jl optimum solution? To keep `sciml_train` consistent

Reply:
yes